src/widget.tsx

```diff
@@ -72,10 +73,24 @@ const App = (props: any): JSX.Element => {
     setSelectedJupyterItems(selectedJupyterItemsTemp);
   };
 
+  const handleLogout = (event) => {
+    event.preventDefault();
+    window.open(getBaseURL('globus-jupyterlab/logout'), 'Logout of Globus', 'height=600,width=800').focus();
+    window.location.reload();
+  };
```

Comment:
This should be a simple JSON call instead of a browser window. Also, `globus-jupyterlab/logout` will only revoke user tokens, but it won't do a full logout of Globus. 

I think it best to first do an API request to `globus-jupyterlab.logout`, then open a browser to `globus.org/logout` to clear out all live sessions with various IdPs.